guides/17_Migration_to_3_0.md

```diff
@@ -53,4 +58,69 @@ handle them please remove.
 - AudioCallEnded
 - TURNMeetingEnded
 - StateMachineTransitionFailed
-- AudioDeviceSwitched
\ No newline at end of file
+- AudioDeviceSwitched
+
+### Updating `EventController` use cases
+We have de-coupled the `EventController` from `AudioVideoController`. Check below for the new additions.
+```typescript
+interface EventController {
+  // Adds an observer for event published to this controller.
+  addObserver(observer: EventObserver): void;
+
+  // Removes an observer for event published to this controller.
+  removeObserver(observer: EventObserver): void;
+
+  // EventReporter that the EventController uses to send events to the Amazon Chime backend.
+  readonly eventReporter?: EventReporter;
+}
+```
+
+Use case 2: Update creation of `EventController`
```

Comment:
This says Use case 2, could not find use case 1. Also, could not find update to:
https://github.com/aws/amazon-chime-sdk-js/pull/1969#discussion_r812463235

Reply:
Added Use Case 1, and line 101 had "or creates one if none is given." added at the end.